relay-server/src/actors/processor.rs

```diff
@@ -1093,9 +1093,7 @@ impl EnvelopeProcessorService {
                 }
             }
             ItemType::ReplayRecording => {
-                // XXX: Temporarily, only the Sentry org will be allowed to parse replays while
-                // we measure the impact of this change.
-                if replays_enabled && state.project_state.organization_id == Some(1) {
+                if replays_enabled {
```

Comment:
This was the change of the original PR. I cherry-picked it in to see the impact on a Canary deploy.